torch/csrc/jit/tensorexpr/codegen.cpp

```diff
@@ -80,6 +82,50 @@ void CodeGen::call_with_numel(void** args, int64_t numel) {
   TORCH_INTERNAL_ASSERT(
       false, "This codegen backend does not implement call_with_numel");
 }
+
+StmtPtr insertMemNodes(std::unordered_set<BufPtr>& interm_bufs, StmtPtr stmt) {
```

Comment:
Nit: Let's be more precise with names. MemNodes can mean many different things, why do we need to change it from `insertAllocFree`?